### DISCUSSION TITLE
Stop publication

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#690] [Q&A] [answered]
A couple of doubts about the stop of a notice:

a) Once the sending of the notice has stopped, it can be sent again, right?
b) When resending the stopped notice, should I keep the same procedureid? And the noticeid (what to do with the version)?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Please get in touch directly with the helpdesk for non technical questions: ted@publications.europa.eu
